src/utils/isValidThemeColorVariant.js

```diff
@@ -1,15 +1,18 @@
-import { themes } from "../theme";
 import { THEME_TM } from "../theme/constants";
+import { getThemeObject } from "./";
 
 const isValidThemeColorVariant = (
-  { themeName = THEME_TM.themeName } = THEME_TM,
+  { themeName = THEME_TM.themeName, customTheme = null } = THEME_TM,
```

Comment:
Would it be cleaner to not nest the `customTheme` values under this `customTheme` key?

E.g. in the event that the `theme` object lacks a `themeName` property, then we can safely assume that the object is a `customTheme` that should be merged with the `globalTheme`.

Reply:
This option gives use the opportunity to have themeName 'lne' for example and overwrite some of the theme's colors. It looks more flexible. What do you think? Do you prefer to go with your solution?

Reply:
No, I agree with you. Can we please just rename `customTheme` to `customValues`? That way we can clarify that we are merging the provided `customValues` in to the theme referenced by the `themeName` passed.